CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Added
+- allow usage of newer `rest-client` version to avoid RPM conflict with sensu-plugins-http
```

Comment:
Any time you move a dependency across majors it will often be a breaking change. Even if its not breaking for this code we have to consider that it will break other gems that have similar issues around exact or strict version constraints in other gems/plugins. This should be under a `### Breaking Changes` header. To be clear the one below should still stay under the existing header that you added.